index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JIC Portfolio</title>
+
+	<link href="https://fonts.googleapis.com/css?family=Playfair+Display:400,700|Poppins" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Roboto:wght@100;700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
+  <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+  <header class="Toolbar">
+    <a href="index.html" class="Mylogo Title2">JIC Software</a>
+    <div class="menu-pc">
+      <a class="item-menu-pc">Portfolio</a>
+      <a class="item-menu-pc">About</a>
+      <a class="item-menu-pc-group">
+      <a class="item-menu-pc">Contact</a>
+        <img class="item-menu-pc-icon" src="images/mail_icon.png" alt="email icon">
+      </a>
+    </div>
+    <nav class="menu-mobile">
+        <img class="burguer-menu" src="images/burguer_menu_icon.png" alt="burguer menu icon">
+    </nav>
+  </header>
+  <section class="main-frame">
+    <div class="content1">
+      <section class="Headline">
+        <h1 class="Primarytext Title1">Hey There.<br>I'm Jorge.</h1>
+        <h2 class="Supportingtext Title2">I'm a Software Developer</h2>
```

Comment:
[OPTIONAL] - Please kindly use the same fonts as the one in the template on [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) design.

According to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md), our website should look exactly like the design (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Let's follow the project core requirements for professional projects, and if we practice from our learning phase, it will be easier in the future at the industry level. Here is an article on [The importance of understanding software requirements](https://www.freecodecamp.org/news/why-understanding-software-requirements-matter-to-you-as-a-software-engineer/)

Your design
![image](https://user-images.githubusercontent.com/78251927/227701355-08956513-66c1-4a4a-8799-f31685252954.png)

Figma
![image](https://user-images.githubusercontent.com/78251927/227701369-dcc85c3c-6c4e-419f-9cd9-46d61f7e8615.png)

Reply:
Done. Thank you @abdulhamiid